planemo/workflow_lint.py

```diff
@@ -57,6 +57,8 @@ class WorkflowLintContext(LintContext):
     # Setup training topic for linting - probably should pass this through
     # from click arguments.
     training_topic = None
+    release = None
```

Comment:
```suggestion
```

this seems unused.

Reply:
I use it to store the release I find into the `.dockstore.yml` to check it matches what is in the workflow.

Reply:
I think it would be better to parse it (again) from the changelog, because one can lint multiple workflows / workflow repos at once and they would use the same lint context. Then things could be messed up.

Reply:
Would you parse it each time you lint a workflow file? Or only once per directory?

Reply:
Whatever is easier to implement and maintain (in code). I think runtime is not important for linting.

Reply:
Sorry, you were write, I defined release here and used version in the code... Anyway I removed both. Thanks.